nozl/nozl.go

```diff
@@ -59,6 +59,7 @@ func SetupNozl(backendPort string) {
 		backendAPIGroup.GET("/service", service.GetServiceAllHandler)
 		backendAPIGroup.GET("/service/:id", service.GetServiceHandler)
 		backendAPIGroup.DELETE("/service/:id", service.DeleteServiceHandler)
+		backendAPIGroup.DELETE("/service", service.DeleteAllServicesHandler)
```

Comment:
Just for the convention we have been following, I would rename this fucntion to
DeleteServiceAllHandler